Sources/SwiftDocCUtilities/ArgumentParsing/Options/DocumentationBundleOption.swift

```diff
@@ -22,7 +22,7 @@ public struct DocumentationBundleOption: ParsableArguments {
     /// The path to a bundle to be compiled by DocC.
     @Argument(
         help: ArgumentHelp(
-            "Path to a documentation bundle directory.",
+            "Path to a documentation archive ('.doccarchive') data directory of JSON files.",
```

Comment:
I'm not sure about the `of JSON files`, I considered:
1. omitting it completely
2. current wording
3. using `containing` instead of `of`